packages/sdk/package.json

```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "dev": "DEV=true node -r esbuild-register helpers/build.ts",
     "build": "node -r esbuild-register helpers/build.ts",
-    "prepublishOnly": "pnpm run build && pnpm run test",
+    "prepublishOnly": "pnpm run build",
```

Comment:
Could we remove this too as we already did a build before this?

Reply:
Out of scope of this PR I would say

Reply:
Fair enough :) I'll come with a followup PR